OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -425,22 +425,29 @@ public static void SetupColorWidget(Widget parent, Session.Slot s, Session.Clien
 			factionFlag.GetImageCollection = () => "flags";
 		}
 
-		public static void SetupEditableTeamWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
+		public static void SetupEditableTeamWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map, string widgetId = "TEAM")
 		{
-			var dropdown = parent.Get<DropDownButtonWidget>("TEAM");
+			var dropdown = parent.Get<DropDownButtonWidget>(widgetId);
+			dropdown.IsVisible = () => true;
 			dropdown.IsDisabled = () => s.LockTeam || orderManager.LocalClient.IsReady;
 			dropdown.OnMouseDown = _ => ShowTeamDropDown(dropdown, c, orderManager, map.PlayerCount);
 			dropdown.GetText = () => (c.Team == 0) ? "-" : c.Team.ToString();
+
+			if (widgetId != "TEAM")
```

Comment:
I am not sure if I should take into account external mods in this phase. I would make it simpler if I didn't want to keep compatibility with external mods.

I think that check is needed - when it is called for host - EditableTemplate, I don't want to hide it. It is hidden only when it is called from NonEditableTemplate with TEAM_DROPDOWN. The TEAM is in boh sections as Label and as Dropdown. I didn't rename it for compatibility.